vars/githubWorkflowRun.txt

```diff
@@ -0,0 +1,69 @@
+Run workflow on github actions
+
+### Run as step:
+
+```
+  def runInfo = githubWorkflowRun(repo: "owner/repository", workflow: "build.yml", ref: "main",
+    parameters: [path: "filebeat"], credentialsId: "github-workflow-token")
+```
+
+### Run asyncronous:
```

Comment:
```suggestion
### Run asynchronous:
```

You can configure the pre-commit locally -> https://github.com/elastic/apm-pipeline-library/blob/master/docs/PRECOMMIT.md#installation

Reply:
fixed